test-suite/objc/DjinniObjcTest.xcodeproj/project.pbxproj

```diff
@@ -597,6 +619,7 @@
 				B5F06A7F1D4973BD005BE736 /* DBObjcOnlyListener.h */,
 				B5F06A801D4973BD005BE736 /* DBObjcOnlyListener+Private.h */,
 				B5F06A811D4973BD005BE736 /* DBObjcOnlyListener+Private.mm */,
+				CFBBB8EC1B6E3A67002EF52F /* DBAccessFlags.h */,
```

Comment:
Compared to the old PR I'm seeing some missing files in here.  Has this been tested?

Reply:
During I resolve the conflict, some of the files are removed in the master branch. 
I did the test by running "make objc" under the test-suite directory and got "** TEST SUCCEEDED **"

Reply:
I found that I really miss 2 header files. I will fix it soon.